src/common/xattr.h

```diff
@@ -19,12 +19,14 @@
 extern "C" {
 #endif
 
+#if defined(__linux__)
```

Comment:
@tchaikov 
I'll put it back in.

But in that case I would be very mich in favour of reversing the whole deal...
Rip out all references to ENODATA and actually start using ENOATTR every where.
Under Linux we then can declare ENODATA = ENOATTR.
But then again, that is perhaps for another set of commits.